WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -77,25 +85,38 @@ struct CartView: View {
             Spacer()
             switch viewModel.orderStage {
             case .building:
-                checkoutButton
-                    .padding(Constants.checkoutButtonPadding)
+                if cartViewModel.isCartEmpty {
+                    EmptyView()
+                }
+                else {
+                    checkoutButton
```

Comment:
Not sure if you want to address it now or later, there's also a divider between items and a button

<img width="275" alt="image" src="https://github.com/user-attachments/assets/6eda1519-0644-45a3-bfd8-d4e12d6c1c1c">

Reply:
Yep, that is a strange shadow that we should, according to figma, show if there are items in that place :D
So I was thinking of adding those shadows later on (that is why I wrote that disclaimer in description of the PR).
 
<img width="1845" alt="Screenshot 2024-07-25 at 15 02 05" src="https://github.com/user-attachments/assets/336173f5-5b7c-46ce-9388-55ebff568fb3">